File: check_db.js
const { MongoClient } = require('mongodb');
require('dotenv').config({ path: '.env.local' });

async function run() {
  const uri = process.env.MONGODB_URI;
  const client = new MongoClient(uri);
  try {
    await client.connect();
    const db = client.db();
    const prod = await db.collection('products').findOne({ trackInventory: true });
    console.log(JSON.stringify(prod, null, 2));
  } finally {
    await client.close();
  }
}
run().catch(console.error);

File: check_db.py
from pymongo import MongoClient
import json
import os

client = MongoClient(os.environ.get("MONGODB_URI", "mongodb+srv://duti:duti@cluster0.zox9u.mongodb.net/dutiheritage?retryWrites=true&w=majority&appName=Cluster0"))
db = client.get_default_database()
prod = db.products.find_one({"trackInventory": True})
if prod:
    prod['_id'] = str(prod['_id'])
    prod['createdAt'] = str(prod['createdAt'])
    prod['updatedAt'] = str(prod['updatedAt'])
    for inv in prod.get('inventory', []):
        if '_id' in inv:
            inv['_id'] = str(inv['_id'])
    print(json.dumps(prod, indent=2))
else:
    print("No products tracking inventory")
